model/models/mistral3/model_text.go

```diff
@@ -21,7 +21,6 @@ type TextOptions struct {
 
 type TextModel struct {
 	model.Base
```

Comment:
This is also bad but I'm not going to change it here since there are complications to the image embeddings in the text model. #10674 addresses those complications so this can be removed afterwards